extern/sector-storage/storiface/worker.go

```diff
@@ -64,9 +75,14 @@ type WorkerJob struct {
 	Hostname string `json:",omitempty"` // optional, set for ret-wait jobs
 }
 
+// CallID uniquely identifies a task submitted by a miner to a worker.
+// It is generated by the worker when it receives a RPC request to perform a task and is returned
+// to the miner as a response to the RPC call so it can use it to track the status of the task.
```

Comment:
```suggestion
// CallID uniquely identifies a task submitted by a miner to a worker.
// It is generated by the worker when it receives a RPC request to perform a task and is returned
// to the miner as a response to the RPC call so it can use it to track the status of the task across node restarts.
```